### PR TITLE
Setup FPGA script

### PR DESCRIPTION
**Pull request by leongross**
If one works with the hardware FPGA some sysfs configurations and kernel module loads have to be performed after each reboot.

This script performs all the required and optional setup steps and adds some sanity checks.

### CODE REVIEW

**Review by MDr164 — COMMENTED** [review]
Can you add a short note in this PR to the FPGA instructions that this script exists? https://github.com/chipsalliance/caliptra-sw/blob/main/hw-latest/fpga/FPGA_Instructions.md

File: setup_fpga.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    fpga_image="$1"
+    if [[ -z "$fpga_image" ]]; then
+        echo "[-] image $fpga_image does not exist. exiting."
+        exit 3
+    fi
+
+    fpgautil -b "$fpga_image" -f Full -n Full
+}
+
+# entryppoint
```
Comment (jhand2) [inline setup_fpga.sh:76]: sp: entrypoint

**Review by jhand2 — APPROVED** [review]

**Review by jlmahowa-amd — CHANGES_REQUESTED** [review]
Thanks for doing this, and sorry for being slow on review.

**Review by MDr164 — APPROVED** [review]

**Comment by leongross** [comment]
@jlmahowa-amd is your change request satisfied? If so, please resolve it so we can merge this.

**Comment by leongross** [comment]
@jlmahowa-amd

**Review by jlmahowa-amd — DISMISSED** [review]

**Review by jhand2 — APPROVED** [review]

**Review by jlmahowa-amd — CHANGES_REQUESTED** [review]
Something went wrong and this no longer reflects the fixes and changes I last reviewed.

**Comment by jlmahowa-amd** [comment]
I pushed a branch with the version of the PR that I tested and approved. Please take a look to see the difference.
https://github.com/chipsalliance/caliptra-sw/compare/main...jlmahowa/fpga_setup

**Comment by jlmahowa-amd** [comment]
These changes were brought in as part of #1270

**Comment by leongross** [comment]
Looks good, thank you for taking care of that. Please excuse the low responsiveness on this.

**Pull request closed without merging**